src/components/Selector/VaultSelector.tsx

```diff
@@ -8,12 +8,27 @@ import { PublicKey } from '../PublicKey';
 interface VaultSelectorProps {
   vaults: ExtendedRegistryVault[];
   selectedVault?: ExtendedRegistryVault;
-  showMaxTokensFor?: 'issuableTokens' | 'redeemableTokens';
+  bridgeDirection?: 'issue' | 'redeem';
```

Comment:
Great change, it is definitely more clear right now ✅ 💪 
What do you think about creating a separate type e.g:
`type BridgeDirection = 'issue' | 'redeem'`

and use it also in the `getMaxTokensForVault` type

Reply:
Sure 👍  I created that enum and put it into the `src/pages/spacewalk/bridge/index.tsx` file. Let me know if you think that works or if there is a better location.